README.md

```diff
@@ -23,31 +24,31 @@ cvc5 is intended to be an open and extensible SMT engine.  It can be used as a
 stand-alone tool or as a library.  It has been designed to increase the
 performance and reduce the memory overhead of its predecessors.  It is written
 entirely in C++ and is released under an open-source software license (see file
-[COPYING](https://github.com/CVC4/CVC4/blob/master/COPYING)).
+[COPYING](https://github.com/cvc5/cvc5/blob/master/COPYING)).
 
 
 Website
 -------------------------------------------------------------------------------
-CVC4's website  is available at:
-http://cvc4.cs.stanford.edu/
+cvc5's website  is available at:
+https://cvc5.github.io/
 
 Documentation
 -------------------------------------------------------------------------------
-Documentation for users of CVC4 is available at:
-http://cvc4.cs.stanford.edu/
+Documentation for users of cvc5 is available at:
+https://cvc5.github.io/docs/
```

Comment:
dito

Reply:
This link actually already has some documentation.